wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/SettingsClientViewController.swift

```diff
@@ -162,7 +172,7 @@ final class SettingsClientViewController: UIViewController,
     }
 
     @objc func onVerifiedChanged(_ sender: UISwitch!) {
-        guard let userSession = ZMUserSession.shared() else { return }
+        guard let userSession = userSession as? ZMUserSession else { return }
```

Comment:
Instead of casting back, it'd be better to add `selfUserClient` to `UserSession`.

Reply:
Done, though should selfUserClient be of type UserType?